app/src/components/ArticlesPublications.jsx

```diff
@@ -23,26 +51,40 @@ class ArticlesPublications extends Component {
     let pageContent = [];
 
     if (articlesPublications && articlesPublications.articles) {
-      const articles = articlesPublications.articles.map(this.renderElement);
+      const publications = articlesPublications.publications.map(this.renderElement);
 
-      if (articles && articles.length) {
```

Comment:
The name? Yes, it less confuse than having two `publications` variables, in my opinion.